README.md

```diff
@@ -56,7 +56,9 @@ for more details.
 
 Make sure you have the following installed:
 
-- [Node.js](https://nodejs.org/en/download/) >= 12
+| Package                                    | Version/-s                                                                             | Link                                  | Note                                                                                                  |
```

Comment:
The EOL for LB4 in the LTS matrix a few lines above needs to be updated as well:

```diff
  | Version    | Status      | Published | EOL                  |
  | ---------- | ----------- | --------- | -------------------- |
- | LoopBack 4 | Current     | Oct 2018  | Apr 2024 _(minimum)_ |
+ | LoopBack 4 | Current     | Oct 2018  | Apr 2025 _(minimum)_ |
  | LoopBack 3 | End-of-Life | Dec 2016  | Dec 2020             |
  | LoopBack 2 | End-of-Life | Jul 2014  | Apr 2019             |
```

Reply:
Solve